src/injectscripts/messages/create.js

```diff
@@ -201,157 +295,164 @@ function DeleteCollection(col) {
 
 }
 
-function LoadCollection(col,cb) {
+function LoadCollection(col, cb) {
     $total = col.items.length;
```

Comment:
is `$total` intended to be a global? If not stick `var` in front of it. And remove the $ prefix because it's not jQuery :P